src/routes/blog/post/How-to-put-privacy-first/+page.markdoc

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: How to create a privacy-first growth strategy
+description: Appwrite is working towards a completely privacy-friendly and open-source tech stack. We are committed to putting developers first and building a company that values your personal data.
+date: 2024-01-12
+cover: /images/blog/privacy.png
+timeToRead: 9
+author: laura-du-ry
+category: GDPR
+---
+
+With our recent announcement of GDPR compliance, we took a big step in becoming a privacy-first organization. But we didn’t want to stop here. We wanted to take it further than just ticking the privacy boxes. We wanted to act as a party that values your personal data, so we needed to take action and move to privacy-friendly analytics. This led to removing most of the tags, pixels, and cookies on our website and the Appwrite Console. In this blog, we dive deeper into what we removed, how this affected our growth strategy, and, most of all, what we, as a community, gained from this decision.
+
+## But first, what are pixels, tags, and cookies?
+
+**Pixels**, **tags**, and **cookies** are all pieces of technology used to [track user activity online.](https://www.osano.com/articles/privacy-cookies-pixels-and-tags) They are often used for marketing and advertising purposes but can also be used to improve website functionality and user experience.
+
+- **Tags** are snippets of code added to a website to track your activity and collect data. For example, a tag can be used to track how many people visit a particular page on a website or to collect information about what products you are viewing. The most well-known tool for this is [Google Tag Manager](https://www.semrush.com/blog/beginners-guide-to-google-tag-manager/). It allows you to manage all of your website tags in one place.
+- **Pixels** are small, transparent images that are embedded in a website. When you visit a website with a pixel, your browser sends information about your visit to the company that owns the pixel. This information can include the user's IP address, browser type, and the URL of the page you’re visiting. It is a beneficial tool to retarget website visitors with banner or social ads.
+- **Cookies,** first-party are set by the website you’ve visited mainly to improve your experience on that website, whereas different websites, such as Google Analytics set third-party cookies. Third-party cookies are the ones that are often used to track you across platforms, mostly meant for advertising purposes.
```

Comment:
Other 2 points sounds like a sentence, but this one reads hard. (beginning)

Reply:
updated!